content/200-concepts/100-components/01-prisma-schema/08-views.mdx

```diff
@@ -347,3 +347,23 @@ const userinfo = await prisma.userInfo.findMany({
 ```
 
 Currently, Prisma Client allows you to update a view if the underlying database allows it, without any additional validation.
+
+## Special types of views
+
+This section describes how to use Prisma with updatable and materialized views in your database.
+
+### Updatable views
+
+Some databases support updatable views (e.g. [PostgreSQL](https://www.postgresql.org/docs/current/sql-createview.html#SQL-CREATEVIEW-UPDATABLE-VIEWS), [MySQL](https://dev.mysql.com/doc/refman/8.0/en/view-updatability.html) and [SQL Server](https://learn.microsoft.com/en-us/sql/t-sql/statements/create-view-transact-sql?view=sql-server-ver16#updatable-views)). Updatable views allow you to create, update or delete entries. 
+
+Currently Prisma treats all `view`s as updatable views. If the underlying database supports this functionality for the view, the operation should succeed. If the view is not marked as updatable, the database will return an error, and Prisma Client will then throw this error.
+
+In the future, Prisma Client might support marking individual views as updatable or not updatable. Please [comment on our `views` feedback issue with your use case](https://github.com/prisma/prisma/issues/17335).
```

Comment:
Make the hyperlink smaller to avoid weird line break:

```suggestion
In the future, Prisma Client might support marking individual views as updatable or not updatable. Please comment on our [`views` feedback issue](https://github.com/prisma/prisma/issues/17335) with your use case.
```

<img width="395" alt="image" src="https://github.com/prisma/docs/assets/4058327/3b4fea93-cd33-4619-bf04-57dbcec39f34">